agent/config/config.go

```diff
@@ -545,6 +545,7 @@ func environmentConfig() (Config, error) {
 		NvidiaRuntime:                       os.Getenv("ECS_NVIDIA_RUNTIME"),
 		TaskMetadataAZDisabled:              utils.ParseBool(os.Getenv("ECS_DISABLE_TASK_METADATA_AZ"), false),
 		CgroupCPUPeriod:                     parseCgroupCPUPeriod(),
+		SpotInstanceDrainingEnabled:         utils.ParseBool(os.Getenv("ECS_SPOT_INSTANCE_DRAINING_ENABLED"), false),
```

Comment:
isn't this going to be true by default?
if yes, then let's have the env var named `ECS_DISABLE_SPOT_INSTANCE_DRAINING` and have default as false.

Reply:
PMs actually want this to be 'false' by default because it's a change in the behavior. In documentation it will be recommended to set to true when using spot instances with ECS.

Reply:
ah ok, I thought it was decided to enable this by default. 
either way, please rename this to `ECS_ENABLE_SPOT_INSTANCE_DRAINING` instead of `ENABLED` as suffix, following the convention - https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-config.html

Reply:
👍